modules/log6x-logging-loki-cli-install.adoc

```diff
@@ -0,0 +1,207 @@
+<4> Specify `redhat-operators` as the value. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object that you created when you configured Operator Lifecycle Manager (OLM).
+
+. Apply the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `namespace` object for deploy the LokiStack:
++
+.Example `namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging # <1>
+  labels:
+    openshift.io/cluster-monitoring: "true" # <2>
+----
+<1> The `openshift-logging` namespace is dedicated for all {logging} workloads.
+<2> A string value that specifies the label, as shown, to ensure that cluster monitoring scrapes the `openshift-logging` namespace.
+
+. Apply the `namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a secret with the credentials to access the object storage. For example, create a secret to access {aws-first} s3.
++
+.Example `Secret` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-s3 # <1>
+  namespace: openshift-logging
+stringData: # <2>
+  access_key_id: <access_key_id>
+  access_key_secret: <access_secret>
+  bucketnames: s3-bucket-name
+  endpoint: https://s3.eu-central-1.amazonaws.com
+  region: eu-central-1
+----
+<1> Use the name `logging-loki-s3` to match the name used in LokiStack.
+<2> For the contents of the secret see the Loki object storage section.
++
+--
+include::snippets/logging-retention-period-snip.adoc[leveloffset=+1]
+--
+
+. Apply the `Secret` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `LokiStack` CR:
++
+.Example `LokiStack` CR
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging # <2>
+spec:
+  size: 1x.small # <3>
+  storage:
+    schemas:
+    - version: v13
+      effectiveDate: "<yyyy>-<mm>-<dd>" # <4>
+    secret:
+      name: logging-loki-s3 # <5>
+      type: s3 # <6>
+  storageClassName: <storage_class_name> # <7>
+  tenants:
+    mode: openshift-logging # <8>
+----
+<1> Use the name `logging-loki`.
+<2> You must specify `openshift-logging` as the namespace.
+<3> Specify the deployment size. Supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`. Additionally, `1x.pico` is supported starting with {logging} 6.1.
+<4> For new installations this date should be set to the equivalent of "yesterday", as this will be the date from when the schema takes effect.
```

Comment:
Does "yesterday" need to be in quotes? 
```suggestion
<4> For new installations, use the date of the previous day, as this will be the date from when the schema takes effect.
```